tests/e2e_tests/kgain_e2e.py

```diff
@@ -26,29 +99,11 @@ def test_l1_to_kgain(tvacdata_path, e2eoutput_path):
         shutil.rmtree(kgain_outputdir)
     os.mkdir(kgain_outputdir)
 
-    # define the raw science data to process
-
-    l1_data_filelist_same_exp = [os.path.join(l1_datadir, "CGI_EXCAM_L1_00000{0}.fits".format(i)) for i in np.arange(51841,51871)]#[51841, 51851]] # just grab some files of it
-    l1_data_filelist_range_exp = [os.path.join(l1_datadir, "CGI_EXCAM_L1_00000{0}.fits".format(i)) for i in np.arange(51731, 51841)]#[51731, 51761]]
-    
     ####### Run the walker on some test_data
-    for file in l1_data_filelist_range_exp:
-        image = data.Image(file)
-        # This should not be necessary anymore after the updates of the OBSTYPE keyword, up to now it is only "SCI"
-        if image.pri_hdr['OBSTYPE'] != 'KGAIN':
-            image.pri_hdr['OBSTYPE'] = 'KGAIN'
-            image.save(filename = file)
-
-    for file in l1_data_filelist_same_exp:
-        image = data.Image(file)
-        # This should not be necessary anymore after the updates of the OBSTYPE keyword, up to now it is only "SCI"
-        if image.pri_hdr['OBSTYPE'] != 'MNFRAME':
-            image.pri_hdr['OBSTYPE'] = 'MNFRAME'
-            image.save(filename = file)
-        l1_data_filelist_range_exp.append(file)
-
-    walker.walk_corgidrp(l1_data_filelist_range_exp, "", kgain_outputdir)
-    kgain_file = os.path.join(kgain_outputdir, "CGI_EXCAM_L1_0000051731_kgain.fits")
+
+    walker.walk_corgidrp(ordered_filelist, "", kgain_outputdir, template="l1_to_kgain.json")
```

Comment:
it would be nice if the variables from the II&T section and the variables from the corgidrp section don't intersect except for clear input/output products. It's not clear what has been done to create `ordered_filelist`. I would actually rather do it again here, rather than reusing the code from II&T.